pages/models.py

```diff
@@ -186,27 +186,111 @@ def get_header_image(self):
 
 
 class HomePage(Page):
-    banner_images = StreamField([
-        ('image', ImageBlock())
-    ], null=True)
-    mobile_banner_images = StreamField([
-        ('image', ImageBlock())
-    ], null=True, blank=True)
-    row_1 = StreamField([
```

Comment:
If `ColumnBlock()` isn't used on another page model, make sure to remove it from `custom_blocks.py`

Reply:
It is used in several other pages